src/main/java/com/ounitech/wemove/models/Member.java

```diff
@@ -36,8 +36,8 @@ public class Member {
     @Column(name = "picture")
     private String picture;
 
-    @Column(name = "active", nullable = false, columnDefinition = "BYTE DEFAULT 0")
-    private Byte active;
+    @Column(name = "active", nullable = false, columnDefinition = "BOOLEAN DEFAULT 0")
```

Comment:
what is the purpose of columnDefintion ? why is it only present in member and not in Subscription and Staff

Reply:
- columnDefintion used to define a default value of zero for  active.
- I forgot to add it to the other entities, i'll include it.